WeScan/Scan/CaptureSessionManager.swift

```diff
@@ -65,6 +65,12 @@ final class CaptureSessionManager: NSObject, AVCaptureVideoDataOutputSampleBuffe
         self.videoPreviewLayer = videoPreviewLayer
         super.init()
         
+        guard AVCaptureDevice.default(for: .video) != nil else {
+            let error = ImageScannerControllerError.inputDevice
```

Comment:
Can we make this error a bit more descriptive, like `ImageScannerControllerError.missingCaptureDevice`?

Reply:
`inputDevice` is defined before i send this pull request, so i use this type.

```
  /// An error occured when setting up the user's device.
    case inputDevice
```

I think this type of error can explain the cause of the error, because it is indeed the user's device that caused the error.